test/common/quic/envoy_quic_client_session_test.cc

```diff
@@ -283,6 +285,11 @@ TEST_P(EnvoyQuicClientSessionTest, ConnectionClose) {
   EXPECT_EQ(absl::StrCat(quic::QuicErrorCodeToString(error), " with details: ", error_details),
             envoy_quic_session_.transportFailureReason());
   EXPECT_EQ(Network::Connection::State::Closed, envoy_quic_session_.state());
+
+  EXPECT_EQ(
+      1U, TestUtility::findCounter(
+              store_, "http3.upstream.rx.quic_connection_close_error_code_QUIC_INVALID_FRAME_DATA")
```

Comment:
do we have docs for this?
I'm not entirely sure where we'd put them.  Currently the upstream stats I'm familiar with are rooted in the cluster so docced in cluster_stats.rst
That said these are only updated when traffic is sent to a cluster, so maybe put them there but make it clear they're not rooted in the cluster and are in fact global?  Unless @mattklein123 or @phlax have better suggestions

Reply:
Yeah I think that might be the best place to put this doc. done.